### ISSUE TITLE
Debian 8 Jessie - 'Sound(.play)' hangs after sound is played. 

### ISSUE THREAD

**Issue by birchb1024** [birchb1024/genyris#15]
Just upgraded to Jessie. Java goes to 100% CPU after playing sound, and hangs up.

**Comment by birchb1024** [comment]
Looks like an openjdk bug: "SourceDataLine.drain() hangs in threaded use" https://bugs.openjdk.java.net/browse/JDK-2038898 ....

**Comment by birchb1024** [comment]
Seems that drain() hangs if it';s called during playback. So poll until its safer....
```
		while( line.isRunning() ){
	        // See Issue https://github.com/birchb1024/genyris/issues/15
		    sleep(100);  // 0.1 second.
		}
	    line.drain(); 
```

**Comment by birchb1024** [comment]
drain() does not hang in openjdk-6 but hangs in openjdk-7

**Comment by birchb1024** [comment]
drain() does NOT hang in openjdk-7 when in the Eclipse debugger.

**Comment by birchb1024** [comment]
Appears to me that Sound is unreliable in Java 7 on Debian. This may never be fixed.

**Comment by birchb1024** [comment]
Rewritten and fixed in ed630182f2f15b3f
